### PR TITLE
Use latex in noise documentation

### PR DESCRIPTION
**Pull request by Strilanc**
- Replace images in docs/noise.md
- Fix docstrings omitting the $$ to activate latex parsing

Fixes https://github.com/quantumlib/Cirq/issues/1552

### CODE REVIEW

**Comment by Strilanc** [comment]
Examples of output:

image

image

**Comment by Strilanc** [comment]
Dave was using Latex, but with a few missteps like `->` instead of `\rightarrow`.

File: docs/noise.md [hunk 1]
```diff
@@ -101,7 +103,7 @@ such that each Pauli gate occurs with probability `p/3`.
 To construct channels, useful helpers are provided `cirq.asymmetric_depolarize`
 and `cirq.depolarize`.
 
-Another common case is when only a Pauli X (bit flip) can occur, or
+Another common case is when only a Pauli X (bit flip) can occur, or()
```
Comment (vtomole) [inline docs/noise.md]: Accidental `()`?
```suggestion
Another common case is when only a Pauli X (bit flip) can occur, or
```

Comment (Strilanc) [reply docs/noise.md]: Oops.

**Pull request merged**